slack-plugin/src/main/kotlin/slack/gradle/tasks/BootstrapTask.kt

```diff
@@ -308,13 +309,34 @@ constructor(objects: ObjectFactory, providers: ProviderFactory) : DefaultTask()
   }
 
   public companion object {
-    public const val NAME: String = "bootstrap"
+    private const val NAME: String = "bootstrap"
 
     /**
      * Current bootstrap version. Other bootstrap tasks can check against this as a minimum version.
      */
     public const val VERSION: Int = 2
 
+    internal fun isBootstrapEnabled(project: Project): Boolean {
+      return project.gradle.startParameter.taskNames.any { it == "bootstrap" }
```

Comment:
Looks like you could use `NAME` here (and on line 330 as well)

Reply:
good point, I'll fix that

Reply:
789a6d6